optimizely/resources/testdatafiles/bucketing_id.json

```diff
@@ -0,0 +1,355 @@
+{
```

Comment:
Let's not introduce a full datafile yet. I'd like to keep the code simpler for now.